metrics/counter.go

```diff
@@ -2,7 +2,7 @@ package metrics
 
 import "sync/atomic"
 
-// Counters hold an int64 value that can be incremented and decremented.
+// Counter methods hold an int64 value that can be incremented and decremented.
```

Comment:
No need for `methods`... it's an interface, just drop the extra `s` and it's fine.

Reply:
Thank you - updated.